### PR TITLE
upgrade crd to apiextensions/v1

### LINKED ISSUE #3708: Upgrade crd to apiextensions.k8s.io/v1
**What's it about?**
upgrade crd to apiextensions.k8s.io/v1

/area upgrade
/kind feature-request

### PR DESCRIPTION
**Pull request by yuswift**
Signed-off-by: yuswift <yuswift2018@gmail.com>

**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
upgrade crd to apiextensions/v1
**Which issue(s) this PR fixes**:
Fixes #3708 

**Special notes for reviewers**:
Globally searched the code and found that only those two files relied on `apiextensions/v1beta` package

### CODE REVIEW

File: pkg/models/resources/v1alpha3/customresourcedefinition/customresourcedefinitions_test.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ import (
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
-	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	v1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
```
Comment (JohnNiang) [inline pkg/models/resources/v1alpha3/customresourcedefinition/customresourcedefinitions_test.go:23]: `extensionV1` may be better than `v1`.

Comment (yuswift) [reply pkg/models/resources/v1alpha3/customresourcedefinition/customresourcedefinitions_test.go:23]: Sure. But this is generated by `goimports`, if we change it to `extensionV1 `, the `Github Action` will fail.

**Pull request merged**